test-avocado/lib/guest-cockpit-redhat.sh

```diff
@@ -30,24 +37,36 @@ if ! rpm -q $BASE_PCKGS >& /dev/null; then
 fi
 
 function yum_install() {
+    echolog "Updating packages"
     yum -y install "$@" | (grep -v "already installed and latest version\|Nothing to do\|Loaded plugins:" || true)
 }
 
 function yum_builddep() {
+    echolog "Updating build dependencies"
     yum-builddep -y "$@" | (grep -v -- "--> Already installed : \|Getting requirements for\|No uninstalled build requires" || true)
 }
 
-echolog "Updating base packages"
-yum_install $BASE_PCKGS
-
-echolog "Updating build dependencies"
-yum_builddep $spec
+function dnf_install() {
+    echolog "Updating packages"
+    dnf -y install "$@" | (grep -v "already installed and latest version\|Nothing to do\|Loaded plugins:" || true)
```

Comment:
yes it is possible, disabling set -e allows to check that and repair it in future. I also found, that dns is unbale to upgrade when use install command, we have to use update instead of install. It does not install latest version